src/modules/70-pipeline/components/PipelineInputSetForm/__tests__/StageInputSetForm.test.tsx

```diff
@@ -412,5 +417,33 @@ describe('stageinputset tests', () => {
       expect(getByText('infrastructureText')).toBeDefined()
       expect(getByText('executionText')).toBeDefined()
     })
+    test('initial render with IACM as stageType', () => {
+      const IACMprops = {
+        deploymentStageTemplate: {
+          serviceConfig: {},
+          execution: {},
+          serviceDependencies: [],
+          infrastructure: {}
+        },
+        deploymentStage: {
+          serviceConfig: {},
+          execution: {},
+          serviceDependencies: [],
+          infrastructure: {}
+        },
+        stageType: StageType.IACM
+      } as any
+
+      const { getByText } = render(
+        <TestWrapper defaultFeatureFlagValues={{ IACM_ENABLED: true }}>
+          <Formik formName="test-form" initialValues={{}} onSubmit={jest.fn()}>
+            <Form>
+              <StageInputSetForm {...IACMprops} viewType={StepViewType.DeploymentForm} />
+            </Form>
+          </Formik>
+        </TestWrapper>
+      )
+      expect(getByText('IACMInputSet')).toBeTruthy()
```

Comment:
Makes zero difference to the result of your test, but `toBeInTheDocument()` conveys what you're trying to prove better than `toBeTruthy()`
```suggestion
      expect(getByText('IACMInputSet')).toBeInTheDocument()
```